content-services/latest/tutorial/sso/ldap.md

```diff
@@ -0,0 +1,296 @@
+
+1. Configuration for LDAP synchronization can be achieved manually for WAR file deployments using the `activiti-ldap-properties` file or reference an external file for Docker and Kubernetes deployments:
+
+    | Property | Description |
+    | -------- | ----------- |
+    | ldap.authentication.enabled | Sets whether LDAP authentication is enabled. This needs to be `false` as LDAP is only being used for user synchronization, for example `false`. |
+    | ldap.authentication.java.naming.provider.url | The URL of the LDAP instance, for example `ldaps://ldap.example.com:636`. |
+    | ldap.synchronization.java.naming.security.principal | The user used to access the LDAP directory to perform the synchronization, for example `uid=admin,ou=system`. |
+    | ldap.synchronization.java.naming.security.credentials | The password for the user set in `ldap.synchronization.java.naming.security.principal`, for example `secret`. |
+    | ldap.synchronization.full.enabled | Sets whether full LDAP synchronization is enabled or not, for example `true`. |
+    | ldap.synchronization.full.cronExpression | The cron expression describing how often the full synchronization should run, for example `0 0 0 * * ?`. |
+    | ldap.synchronization.differential.enabled | Sets whether differential LDAP synchronization is enabled or not, for example `true`. |
+    | ldap.synchronization.differential.cronExpression | The cron expression describing how often the differential synchronization should run, for example `0 0 */4 * * ?`. |
+    | ldap.synchronization.userSearchBase | The section of the LDAP directory to restrict user synchronization to, for example `ou=users,dc=alfresco,dc=com`. |
+    | ldap.synchronization.groupSearchBase | The section of the LDAP directory to restrict group synchronization to, for example `ou=groups,dc=alfresco,dc=com`. |
+
+2. Configuration with the Alfresco Process Services can be achieved manually for WAR file deployments using the `activiti-identity-service.properties` or reference an external file for Docker and Kubernetes deployments:
+
+    | Property | Description |
+    | -------- | ----------- |
+    | keycloak.enabled | Sets whether Process Services will use Keycloak to authenticate against, for example `true`. |
+    | keycloak.realm | The realm name configured in Keycloak for the Alfresco applications, for example `alfresco`. |
+    | keycloak.auth-server-url | Keycloak's base URL, for example `https://keycloak.example.com/auth`. |
+    | keycloak.ssl-required | Sets whether SSL is mandatory for access or not, for example `all`. |
+    | keycloak.resource | The **Client ID** set up in Keycloak for Process Services. The client needs to exist underneath the realm set for `keycloak.realm` or `KEYCLOAK_REALM`, for example `alfresco`. |
+    | keycloak.principal-attribute | The attribute to identify users by for authentication. This needs to be set to `email` for Process Services, for example `email`. |
```

Comment:
**Is there a DOCS ticket for this change? Why hasn't it been reflected into the official SSO guide yet?** 

**Please coordinate with and guide @mbujak-hyland through the necessary changes** that this will require both for the "latest" (23.x) version of ACS and for previous ones depending on their supported platform's stated supported version of APS. E.g.: if ACS 7.3 and ACS 7.4 do not list APS 24.x as supported platform then the properties as they were in APS 2.x should be part of the docs there. Whereas for the latest we should have the new APS docs.